reference/robot.md

```diff
@@ -191,13 +191,12 @@ This function synchronizes the sensor and actuator data between Webots and the
 controllers. If the `wb_robot_step()` function is not called then there will be
 no actuation in Webots and no update of the sensors in the controller.
 
-The `ms` parameter specifies the number of milliseconds that must be simulated
+The `duration` parameter specifies the amount of time, expressed in milliseconds, that must be simulated
 until the `wb_robot_step()` function returns. Note that this is not real time
 but virtual (simulation) time, so this is not like calling the system's
-`sleep()`. In fact the function may return immediately, however the important
-point is that when it returns `ms` milliseconds of simulation will have elapsed.
-In other words the physics will have run for `ms` milliseconds and hence the
-motors may have moved, the sensor values may have changed, etc. Note that `ms`
+`sleep()`. Depending on the complexity of the simulation and execution mode, the function may return quickly. When it returns, the requested duration of simulation time is elapsed.
+In other words the physics will have run for the specified duration and hence the
```

Comment:
I think that now the verb tense 'will have run' doesn't make sense any longer.
Maybe simply 'is run'  or 'runs' is better.